### ISSUE TITLE
Regression algorithm returns many negative values

### ISSUE THREAD

**Issue by RonBig** [catboost/catboost#1357]
When I run Catboost regression algorithm it predict negative values for many, sometimes most of my records.

In my training data there is not a single point in which we provide negative value (There are lots of 0s, though)

How is that possible? 
Also, how I can reduce it drastically?

I thought about using a custom loss function, but it does not work at the moment ....

**Comment by kizill** [comment]
Hello! Sorry for a long delay.
CatBoost is a classical boosting algorithm and thus sum individual regression tree predictions could be out of [min_value, max_value] of training set. Custom loss function could help you, what's the problem with it?

**Comment by kizill** [comment]
Hello! Gently remiding you to share your problems with providing custom loss functions :)

**Comment by RonBig** [comment]
hi
I've been told they do not work at the moment in this post
https://github.com/catboost/catboost/issues/1300#issuecomment-657405894
